code-for-canada/project-thundercat
# [Accessibility] Fixing quit test label

# Description

This PR is simply fixing the "Quit test" button label. According to David, when selecting this button, the screen reader should say "Quit test" and not the description of the action itself like it does now.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshot

(N/A)

# Testing

1.  Launch NVDA.
2.  Start sample test.
2.  Navigate using the keyboard and focus on "Quit test" button and make sure that the screen reader says "Quit test button".

# Checklist
